bridge/contracts/Snapshots.sol

```diff
@@ -157,25 +157,25 @@ contract Snapshots is Initializable, SnapshotsStorage, ISnapshots {
         returns (bool)
     {
         {
-            require(
-                _epoch == 0,
-                string(abi.encodePacked(SnapshotsErrorCodes.SNAPSHOT_MIGRATION_NOT_ALLOWED))
-            );
-            require(
-                groupSignature_.length == bClaims_.length && groupSignature_.length >= 1,
-                string(abi.encodePacked(SnapshotsErrorCodes.SNAPSHOT_MIGRATION_INPUT_DATA_MISMATCH))
-            );
+            if (_epoch != 0) {
+                revert SnapshotsErrors.MigrationNotAllowedAtCurrentEpoch();
+            }
+            if (!(groupSignature_.length == bClaims_.length && groupSignature_.length >= 1)) {
```

Comment:
What do you think about reverting the boolean expression for more clarity? 
```
groupSignature_.length != bClaims_.length || groupSignature_.length < 1
```

Reply:
There's quite a few instances where i wrapped the parameters of the old `require` statements and checked for the reverse given that i was converting to `if` statements and I wanted to ensure the logic wasn't broken. I'll go through the instances of this and refactor for readability 👍